labs/module4/English/Fortran/README.ipynb

```diff
@@ -114,7 +114,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pgfortran -fast -ta=multicore -Minfo=accel -o laplace_multicore laplace2d.f90 jacobi.f90 && ./laplace_multicore"
+    "!pgf90 -fast -ta=multicore -Minfo=accel -o laplace_multicore laplace2d.f90 jacobi.f90 && ./laplace_multicore"
```

Comment:
This should use `pgfortran`, `pgf90` is deprecated.

Reply:
Fixed and create new pull request.